impeller/entity/contents/solid_rrect_blur_contents.cc

```diff
@@ -71,10 +71,15 @@ bool SolidRRectBlurContents::Render(const ContentContext& renderer,
 
   VertexBufferBuilder<VS::PerVertexData> vtx_builder;
 
-  // Clamp the max kernel width/height to 1000 to limit the extent
+  Matrix basis_invert = entity.GetTransform().Basis().Invert();
+  Vector2 max_sigmas =
+      Vector2((basis_invert * Vector2(250.f, 0.f)).GetLength(),
+              (basis_invert * Vector2(0.f, 250.f)).GetLength());
```

Comment:
This seems like a lot of calculations. Basically don't we want to avoid the case of the X sigma being scaled to more than 250 and same with the Y? We should be able to directly calculate the expansion factor of the matrix in both dimensions and then divide by that expansion factor. I think GetMaxBasisLengthXY() might do the trick?

Reply:
I reverted this since it's not quite the same.  I'm pretty sure I want the min of those values not the max.

Reply:
You didn't do what I asked, but did something else entirely. I'm suggesting that you divide the largest sigma by the MaxBasisLength() - there is never any matrix inversion in that concept. Something like:

```
  // Pseudo code as I don't think we have scalar/vector division defined.
  max_sigmas = 250 / entity.GetTransform().MaxBasisLengthXY();
  sigmas = min(sigmas, max_sigmas);
```

Reply:
That's why it was inverted.  An inverted multiply is the same thing as a divide.  That wasn't the issue though, MaxBasisLengthXY() returns one value and it's not the one we want.  I want the minimum.

Reply:
It's even more complicated because I'm looking for the BasisLengths as a vector, but the methods only return a scalar. So perhaps something like this:

```
  transform = entity.GetTransform();
  max_sigmas = {250 / transform.GetXBasisLength(), 250 / transform.GetYBasisLength()};
  sigmas = min(sigmas, max_sigmas);
```

We'd need the individual basis lengths from the transform, either delivered in a pair of Scalar methods, or in a method that returns a vector of both rather than the min/max of both.

Reply:
> That's why it was inverted. An inverted multiply is the same thing as a divide. That wasn't the issue though, MaxBasisLengthXY() returns one value and it's not the one we want. I want the minimum.

An inversion takes almost 200 multiplies and 80 adds. It's not the same.

Reply:
MaxBasis takes the elements representing how much X distances and Y distances are expanded and finds the max of them. In other words, what is the biggest expansion factor we have for how big a distance will be expanded in either X or Y.

250 divided by that value is the largest user space sigma we can have that will be expanded to no more than 250 in device space once it is transformed.

Reply:
Why do you want the min? Is it because you only want to clamp in the smaller direction and let the other direction scale as much as it wants - to infinity?

Reply:
Ah, yes, if you are doing `Invert.MaxBasis` then you are getting the max of the inverted values, but after inversion you want the min. But Invert is a really expensive operation in the first place and you are missing the MinBasis function you would want.

But, 1/MaxBasis uses the "max" that already exists, does it before the multiplicative inverse, so the computations work out to what they should have been if you used Invert.MinBasis and are much faster.